Add HumanLLM

amicus-veritatis opened the pull request:
This PR creates HumanLLM, a simple LLM wrapper class that returns user input as the response.

I believe this LLM wrapper will be useful especially for debugging, educational or testing purpose.

amicus-veritatis committed:
added human_llm

amicus-veritatis committed:
fixed typo

amicus-veritatis committed:
Merge branch 'hwchase17:master' into humanllm

amicus-veritatis committed:
Minor typo

amicus-veritatis committed:
Minor typo

hwchase17 reviewed (commented):
lets 

(1) change this naming to be "testing" or something

(2) add a notebook/documentation for how and why to use

hwchase17 committed:
Harrison/one drive loader (#4081)

Co-authored-by: José Ferraz Neto <netoferraz@gmail.com>

hwchase17 committed:
Harrison/one drive loader (#4081)

Co-authored-by: José Ferraz Neto <netoferraz@gmail.com>

amicus-veritatis committed:
Merge branch 'hwchase17:master' into humanllm

vowelparrot committed:
Vwp/sqlalchemy (#4145)

Bump threshold to 1.4 from 1.3. Change import to be compatible

Resolves #4142 and #4129

---------

Co-authored-by: ndaugreal <ndaugreal@gmail.com>
Co-authored-by: Jeremy Lopez <lopez86@users.noreply.github.com>

amicus-veritatis committed:
Changed the name into HumanInputLLM

amicus-veritatis committed:
Changed the name into HumanInputLLM

amicus-veritatis committed:
Changed the name into HumanInputLLM

amicus-veritatis committed:
Merge branch 'hwchase17:master' into humanllm

amicus-veritatis committed:
Merge branch 'humanllm' of https://github.com/amicus-veritatis/langchain into humanllm

amicus-veritatis committed:
docs

amicus-veritatis committed:
docs

amicus-veritatis committed:
minor change

amicus-veritatis committed:
docs

amicus-veritatis committed:
Merge branch 'hwchase17:master' into humanllm

amicus-veritatis commented:
@hwchase17 I opened #4160 since (1) I was unable to unstage mistakenly committed files (poetry.lock and pyproject.toml), (2) I felt closing this PR and operning new PR would be more appropriate if I changed class name.
